src/stores/queueStore.ts

```diff
@@ -213,7 +241,7 @@ export class TaskItemImpl {
       : undefined
   }
 
-  public async loadWorkflow() {
+  public async loadWorkflow(app: ComfyApp) {
```

Comment:
It's wield that if I don't import `plainToClass` from class transformer, vitest is throwing reflect-metadata not available error when import `app.ts`. I think it's time we just remove `app.ts` from the import list and accept it as a param from caller.